localgov_demo.install

```diff
@@ -11,40 +11,39 @@ use Drupal\menu_link_content\Entity\MenuLinkContent;
  * Implements hook_install().
  */
 function localgov_demo_install($is_syncing) {
-  if ($is_syncing) {
-    return;
-  }
-
-  // Enable area geo bundle in directory venue content type..
-  $localgov_directories_venue_config = \Drupal::service('config.factory')->getEditable('field.field.node.localgov_directories_venue.localgov_location');
-  $localgov_directories_venue_config->set('settings.handler_settings.target_bundles.area', 'area');
-  $localgov_directories_venue_config->save();
-
-  // Set demo front page.
-  $system_site = \Drupal::configFactory()->getEditable('system.site');
-  $system_site->set('page.front', '/localgov-drupal-demo');
-  $system_site->save();
-  drupal_flush_all_caches();
 
-  // Disable unwanted blocks from localgov_base and localgov_scarfolk.
-  $blocks_to_disable = [
-    'block.block.views_block__services_block_service_list_base',
-    'block.block.views_block__services_block_service_list_scarfolk',
-    'block.block.localgov_home_welcome_block_base',
-    'block.block.localgov_home_welcome_block_scarfolk',
-  ];
-  foreach ($blocks_to_disable as $block_id) {
-    // Load block and set disabled.
-    $services_home_block = \Drupal::configFactory()->getEditable($block_id);
-    $services_home_block->set('region', 'disabled');
-    $services_home_block->save();
+  if (!$is_syncing) {
+    // Enable area geo bundle in directory venue content type..
+    $localgov_directories_venue_config = \Drupal::service('config.factory')->getEditable('field.field.node.localgov_directories_venue.localgov_location');
+    $localgov_directories_venue_config->set('settings.handler_settings.target_bundles.area', 'area');
+    $localgov_directories_venue_config->save();
+
+    // Set demo front page.
+    $system_site = \Drupal::configFactory()->getEditable('system.site');
+    $system_site->set('page.front', '/localgov-drupal-demo');
+    $system_site->save();
+    drupal_flush_all_caches();
+
+    // Disable unwanted blocks from localgov_base and localgov_scarfolk.
+    $blocks_to_disable = [
+      'block.block.views_block__services_block_service_list_base',
+      'block.block.views_block__services_block_service_list_scarfolk',
+      'block.block.localgov_home_welcome_block_base',
+      'block.block.localgov_home_welcome_block_scarfolk',
+    ];
+    foreach ($blocks_to_disable as $block_id) {
+      // Load block and set disabled.
+      $services_home_block = \Drupal::configFactory()->getEditable($block_id);
+      $services_home_block->set('region', 'disabled');
+      $services_home_block->save();
+    }
+
+    // Set weight of localgov_alert_banner block to float up above the banner.
+    $alert_banner_block = \Drupal::configFactory()->getEditable('block.block.localgov_alert_banner');
+    $alert_banner_block->set('weight', '-10');
+    $alert_banner_block->save();
```

Comment:
Lets be careful not to re-enable this, maybe rebase from 3.x?

Reply:
I doubt it'll rebase as it's indenting all the code.

But the main code has updated. Lemme just make it again.